driver-service/src/test/java/com/dulfinne/taxi/driverservice/unit/service/CarServiceTest.java

```diff
@@ -0,0 +1,232 @@
+package com.dulfinne.taxi.driverservice.unit.service;
+
+import com.dulfinne.taxi.driverservice.dto.request.CarRequest;
+import com.dulfinne.taxi.driverservice.dto.response.CarResponse;
+import com.dulfinne.taxi.driverservice.exception.EntityAlreadyExistsException;
+import com.dulfinne.taxi.driverservice.exception.EntityNotFoundException;
+import com.dulfinne.taxi.driverservice.mapper.CarMapper;
+import com.dulfinne.taxi.driverservice.model.Car;
+import com.dulfinne.taxi.driverservice.repository.CarRepository;
+import com.dulfinne.taxi.driverservice.service.impl.CarServiceImpl;
+import com.dulfinne.taxi.driverservice.util.CarTestData;
+import com.dulfinne.taxi.driverservice.util.PaginationTestData;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mapstruct.factory.Mappers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class CarServiceTest {
```

Comment:
Нет необходимости делать тестовые классы public.
```suggestion
class CarServiceTest {
```